src/core/utils.js

```diff
@@ -78,7 +78,7 @@ export function sanitizeHTML(text, allowTags = []) {
 
 // Convert new lines to HTML breaks.
 export function nl2br(text) {
-  return text.replace(/(?:\r\n|\r|\n)/g, '<br />');
+  return (text || '').replace(/(?:\r\n|\r|\n)/g, '<br />');
```

Comment:
it would be great to get a test added for this: https://github.com/mozilla/addons-frontend/blob/master/tests/client/core/test_utils.js#L318